custom_components/robonect/client/client.py

```
@@ -76,13 +76,32 @@
+            command: The command string to send to the mower. If None, returns False.
+            params: Optional dictionary of parameters to include in the request.
+        
+        Returns:
+            The parsed JSON response from the mower as a dictionary, or a transformed
+            single-depth dictionary if enabled. For certain commands, returns a success
+            dictionary. Returns False if the response is not valid JSON.
+        
+        Raises:
+            Exception: If no response is received from the mower.
+            httpx.HTTPStatusError: If the mower returns an HTTP error status code.
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)